grammar.js

```diff
@@ -820,7 +827,7 @@ module.exports = grammar({
       optional(field('default_value', $._labeled_parameter_default_value)),
     ),
 
-    type_parameter: $ => seq(
+    abstract_type: $ => seq(
```

Comment:
Renaming because there is a rule with similar name `type_parameters`.

In OCaml it is [astract_type](https://github.com/tree-sitter/tree-sitter-ocaml/blob/master/ocaml/grammar.js#L811-L814)